zerver/lib/integrations.py

```diff
@@ -0,0 +1,128 @@
+from typing import Dict, List, Optional
+from django.conf import settings
+from django.conf.urls import url
+
+
+class Integration(object):
+    DEFAULT_LOGO_STATIC_PATH = 'static/images/integrations/logos/{name}.png'
```

Comment:
The logos don't currently work for me; I think the problem is you need a leading slash here.